boards/esp32-ethernet-kit-v1_0/Makefile.features

```diff
@@ -0,0 +1,17 @@
+CPU_MODEL = esp32-wrover
+
+# common board and CPU features
+include $(RIOTBOARD)/common/esp32/Makefile.features
+
+# additional features provided by the board
+FEATURES_PROVIDED += periph_adc
+FEATURES_PROVIDED += periph_i2c
+FEATURES_PROVIDED += periph_pwm
+
+ifeq (,$(filter esp_jtag,$(USEMODULE)))
+  FEATURES_PROVIDED += periph_spi
+endif
+
+# unique features provided by the board
+FEATURES_PROVIDED += esp_jtag
```

Comment:
`HAS_ESP_JTAG` has to be defined. Since several ESP32 boards can have a JTAG interfacem, it should be defined in `cpu/esp_common/Kconfig`.

Reply:
Added JTAG for the esp32-wrover-kit as well.